content/src/main/content/jcr_root/apps/pokegolure/components/manage/clientlibs/js/script.js

```diff
@@ -40,6 +43,10 @@ PokeGoLure.Manage = (function ($) {
         $signOutBtn.on('click', _handleSignOut);
         $luresList.on('click', '.pokego-manage__lures__item__delete', _handleLureDeleteClick);
         $luresList.on('mouseenter mouseleave', '.pokego-manage__lures__item', _handleLureMouseMovement);
+
+        _fetchPokeData();
+
+        setInterval(_fetchPokeData, 1000 * 60); // Fetch data every 60 seconds
```

Comment:
i dont know about this polling. Might interfere with the scheduled service which is also changing location? Users can just refresh the page if they want an updated count I reckon.